scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -262,9 +268,17 @@ build-provider-create $bartender_name
     cp -aR "$LIVECD_ROOTFS_DIR" $temp_dir/livecd-rootfs
   fi
 
+  if [ -n "$UBUNTU_OLD_FASHIONED_DIR" ]
```

Comment:
Why not make this logic an if-else rather than two mutually-exclusive ifs?

Reply:
they are mutually exclusive ifs. We could use local directories for all of these things at the same time (e.g. custom livecd-rootfs dir, custom hooks dir, custom old-fashioned dir, etc.). Or we could use a subset, or none at all.

Reply:
Isn't `if [ -n "$UBUNTU_OLD_FASHIONED_DIR" ]` mutually exclusive with `if [ -z "$UBUNTU_OLD_FASHIONED_DIR" ]` below?

Why not make this an if-else?

Reply:
@codyshepherd Oh, sorry, I misunderstood your comment. Yes, that's a good suggestion, but will require a little bit of rework. Mind if I open another PR after this merges to address that?